src/config/dynamic-theme-fixes.config

```diff
@@ -33101,6 +33101,16 @@ tp-yt-paper-menu-button {
 #paid-comment-background.ytd-comment-view-model {
     background-color: transparent !important;
 }
+svg[id^="yt-ringo2"] :not([id^="youtube-paths"]) path:nth-child(1) {
+    fill: #ff0033 !important;
+}
+ytd-horizontal-card-list-renderer.ytd-structured-description-content-renderer:not(:first-child),
+ytd-metadata-row-container-renderer.ytd-structured-description-content-renderer:not(:first-child),
+ytd-structured-description-content-renderer[inline-structured-description] ytd-horizontal-card-list-renderer.ytd-structured-description-content-renderer,
+ytd-structured-description-content-renderer[inline-structured-description] ytd-error-corrections-section-renderer.ytd-structured-description-content-renderer,
+ytd-structured-description-content-renderer[inline-structured-description] ytd-video-description-infocards-section-renderer.ytd-structured-description-content-renderer {
+    border-top: 2px solid #222 !important;
```

Comment:
Have you checked how this would look like using light scheme in Dark Reader?
Maybe we should use `${#DDD}` instead to have it light in light scheme and dark in dark?